plugins/gfm/gfm-all-module-page/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+import org.jetbrains.registerDokkaArtifactPublication
+
+dependencies {
+    implementation(project(":plugins:base"))
+    implementation(project(":plugins:gfm"))
+    implementation(project(":plugins:all-module-page"))
+}
+
+registerDokkaArtifactPublication("dokkaGfmAllModulePage") {
+    artifactId = "gfm-all-module-page-plugin"
```

Comment:
shouldn't we just create a new `TemplateProcessingStrategy` for GFM and choose them depending of the input file format? This way we wouldn't force users to create 2 plugins instead of one if they want to have a multimodule.

Reply:
Yes, this is not yet a full generation for gfm. We still need additional plugin due to a conflicts in location providers.